checkov/terraform/checks/graph_checks/gcp/GCRContainerVulnerabilityScanningEnabled.yaml

```diff
@@ -0,0 +1,27 @@
+metadata:
+  id: "CKV2_GCP_11"
+  name: "GCP GCR Container Vulnerability Scanning is enabled"
```

Comment:
would be great to rephrase it with `Ensure ...`

Reply:
suggestion applied